extensions/amp-form/0.1/validation-bubble.js

```diff
@@ -51,10 +51,10 @@ export class ValidationBubble {
     this.isVisible_ = false;
 
     /** @private @const {!Element} */
-    this.bubbleElement_ = win.document.createElement('div');
+    this.bubbleElement_ = ampdoc.win.document.createElement('div');
     this.bubbleElement_.classList.add('-amp-validation-bubble');
     this.bubbleElement_[OBJ_PROP] = this;
-    win.document.body.appendChild(this.bubbleElement_);
+    ampdoc.win.document.body.appendChild(this.bubbleElement_);
```

Comment:
I was just trying to transpose the code to use the `ampdoc.win`, but I don't have a good reason in mind to not use `ampdoc.getBody()`. The tests pass with `ampdoc.getBody()`, do you see a reason to use the global document body?